tests/test_vehicle_registry.py

```
@@ -193,6 +214,914 @@ def test_paused_vehicle_requires_reason(self):
+            self.assertIn("PAUSED", second_call)
+            self.assertIn("vehicle_b", second_call)
+            self.assertIn("test pause", second_call)
+
+    def test_main_default_validate_action_prints_summary(self):
+        with tempfile.TemporaryDirectory() as directory:
+            root = Path(directory)
+            (root / "config.json").write_text(json.dumps(governed_config("test_vehicle")))
+            registry = minimal_registry([registry_entry("test_vehicle", "config.json")])
+            (root / "vehicle_registry.json").write_text(json.dumps(registry))
+            with patch("vehicle_registry.Path.cwd", return_value=root):
+                with patch("builtins.print") as mock_print:
```

Ruff (0.16.1)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)